examples/common_patterns_for_connectors/export/csv/connector.py

```diff
@@ -0,0 +1,114 @@
+# This is a simple example of how to work with CSV file response for a REST API of export type.
+# It defines a simple `update` method, which upserts retrieved data to a table named "item".
+# THIS EXAMPLE IS TO HELP YOU UNDERSTAND CONCEPTS USING DUMMY DATA. IT REQUIRES THE FIVETRAN-API-PLAYGROUND PACKAGE
+# (https://pypi.org/project/fivetran-api-playground/) TO RUN.
+# See the Technical Reference documentation
+# (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details.
+
+
+# Import requests to make HTTP calls to API.
+import requests as rq
+# Import CSV for parsing CSV data.
+import csv
+# Import required classes from fivetran_connector_sdk.
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Logging as log
+from fivetran_connector_sdk import Operations as op
+from io import StringIO
+
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function:
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+def schema(configuration: dict):
+    return [
+        {
+            "table": "user",
+            "primary_key": ["id"],
+            "columns": {
+                "id": "STRING",
+                "name": "STRING",
+                "email": "STRING",
+                "address": "STRING",
+                "company": "STRING",
+                "job": "STRING",
+                "updatedAt": "UTC_DATETIME",
+                "createdAt": "UTC_DATETIME",
+            },
+        }
+    ]
+
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector.
+# - state: a dictionary that contains whatever state you have chosen to checkpoint during the prior sync.
+# The state dictionary is empty for the first sync or for any full re-sync.
+def update(configuration: dict, state: dict):
+    log.warning("Example: Handling CSV Response - Syncing data from CSV Export API")
```

Comment:
Why warning?

Reply:
this is product's decision to easily identify all example connectors, we have the same for other pagination examples.